lib/Network/OAuth2/Server/API.hs

```diff
@@ -7,6 +23,14 @@
 {-# LANGUAGE RecordWildCards     #-}
 {-# LANGUAGE TypeOperators       #-}
 
+-- | Anchor specific OAuth2 implementation.
```

Comment:
It isn't all that Anchor-specific excepting "shibboleth" and that's really just "auth by a front-end proxy"; `mod_ldap` can do/does the same thing we expect.